packages/api-headless-cms/src/crud/contentEntry/useCases/RestoreEntryFromBin/TransformEntryRestore.ts

```diff
@@ -0,0 +1,68 @@
+import { SecurityIdentity } from "@webiny/api-security/types";
+import { entryFromStorageTransform, entryToStorageTransform } from "~/utils/entryStorage";
+import { getDate } from "~/utils/date";
+import { getIdentity } from "~/utils/identity";
+import { validateModelEntryDataOrThrow } from "~/crud/contentEntry/entryDataValidation";
+import { CmsContext, CmsEntry, CmsEntryStorageOperationsMoveToBinParams, CmsModel } from "~/types";
+
+export class TransformEntryRestore {
```

Comment:
`TransformEntryRestoreFromBin` maybe?

I saw you have the `RestoreFromBin` everywhere, so just dbl checking if we want to do the same here.

Reply:
Done 👍 
Thank you!